cli/blkio.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+	units "github.com/docker/go-units"
+)
+
+// WeightDevice defines weight device
+type WeightDevice struct {
+	values []*types.WeightDevice
+}
+
+func validateWeightDevice(val string) (*types.WeightDevice, error) {
+	pairs := strings.Split(val, ":")
+	if len(pairs) != 2 {
+		return nil, fmt.Errorf("bad format for weight device: %s", val)
+	}
+
+	weight, err := strconv.ParseUint(pairs[1], 10, 0)
+	if err != nil {
+		return nil, fmt.Errorf("invalid weight for device: %s", val)
+	}
+	if weight > 0 && (weight < 10 || weight > 1000) {
+		return nil, fmt.Errorf("invalid weight for device: %s", val)
```

Comment:
fmt.Errorf("invalid weight for device %s: weight should be in [10, 1000], or 0 to disable", val)